### PR TITLE
feat: populate EventJobSummary with v2 data

### PR DESCRIPTION
**Pull request by yesnault**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-cds#20643.0 ✘  
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20644.0 ✘  
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20645.0 ✘  
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20645.1 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20645.1 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✘  
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20645.2 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✘  
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20645.3 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✘  
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20645.4 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20657.0 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✘  
  * Standalone Tests v2 ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20664.0 ✘  
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20665.0 ✘  
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Review by richardlt — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20665.1 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20672.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20675.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Review by sguiheux — APPROVED** [review]

**Review by sguiheux — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#20679.0 ✘  
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20679.1 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20679.1 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

File: engine/cache/redis.go [hunk 1]
```diff
@@ -273,7 +274,7 @@ func (s *RedisStore) DequeueWithContext(c context.Context, queueName string, wai
 			}
 			if err == nil && len(res) == 2 {
 				elem = res[1]
-				break
+				break forloop
```
Comment (fsamin) [inline engine/cache/redis.go:277]: useless

**Review by fsamin — CHANGES_REQUESTED** [review]

**Review by fsamin — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#20682.0 ✘  
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#20682.1 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20682.1 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20682.2 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**